exp-models/addon/models/experiment.js

```diff
@@ -120,6 +127,7 @@ export default DS.Model.extend(JamModel, {
         var collection = this.store.createRecord('collection', {
             id: 'experimenter.' + this.get('sessionCollectionId')
```

Comment:
Instead of a hardcoded value, can you change this to use the `ENV.JAMDB.namespace` setting?